units/breakpoint-pre-mount.service.in

```diff
@@ -0,0 +1,38 @@
+#  SPDX-License-Identifier: LGPL-2.1-or-later
+#
+#  This file is part of systemd.
+#
+#  systemd is free software; you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as published by
+#  the Free Software Foundation; either version 2.1 of the License, or
+#  (at your option) any later version.
+
+[Unit]
+Description=Breakpoint before mounting the root filesystem on /sysroot
+Documentation=man:systemd-debug-generator(8)
+AssertPathExists=/etc/initrd-release
+DefaultDependencies=no
+Conflicts=shutdown.target emergency.target
+After=sysinit.target systemd-vconsole-setup.service
+Before=initrd-root-fs.target sysroot.mount systemd-fsck-root.service
+
+[Service]
+Environment=TERM=linux
+Environment=PS1="pre-mount:$PWD# "
```

Comment:
hmm, i think we should instead set SHELL_PROMPT_PREFIX here, making use of 229d4a980607e9478cf1935793652ddd9a14618b

Reply:
I also though so but at this point we are in the initrd and there might not be the profile.d file which consumes that variable

Reply:
i think that's reason to include it in the initrd then...